pkg/verifier/package_verifier.go

```diff
@@ -28,18 +29,19 @@ type verifierService interface {
 type ValidateEgressInput struct {
 	// Timeout sets the maximum duration an egress endpoint request can take before it aborts and
 	// is retried or marked as blocked
-	Timeout                              time.Duration
-	Ctx                                  context.Context
-	SubnetID, CloudImageID, PlatformType string
-	EgressListYaml                       string
-	Proxy                                proxy.ProxyConfig
-	Tags                                 map[string]string
-	AWS                                  AwsEgressConfig
-	GCP                                  GcpEgressConfig
-	SkipInstanceTermination              bool
-	TerminateDebugInstance               string
-	ImportKeyPair                        string
-	ForceTempSecurityGroup               bool
+	Timeout                 time.Duration
+	Ctx                     context.Context
+	SubnetID, CloudImageID  string
+	PlatformType            cloud.Platform
```

Comment:
Let's add a docstring here similar to `Timeout`. Please mention the default value (AWSClassic)